src/parser.jl

```diff
@@ -50,15 +50,15 @@ end
 Parser(input::String) = Parser(IOBuffer(input))
 Base.error(p::Parser, l, h, msg) = push!(p.errors, ParserError(l, h, msg))
 Base.eof(p::Parser) = eof(p.input)
-Base.position(p::Parser) = position(p.input)+1
+Base.position(p::Parser) = Int(position(p.input))+1
 Base.read(p::Parser) = read(p.input, Char)
 
 "Rewind parser input on `n` characters."
 function rewind(p::Parser, n=1)
     pos = position(p.input)
     pos == 0 && return 0
     skip(p.input, -n)
-    return position(p.input)
+    return Int(position(p.input))
```

Comment:
Why is this a correct change?  If `p.input` is a large file on a 32-bit machine, won't this fail for files > 2GB?